cwms-data-api/src/main/java/cwms/cda/data/dao/LocationsDaoImpl.java

```diff
@@ -131,6 +133,29 @@ public List<Location> getLocations(String nameRegex, String unitSystem, String d
                     .fetch(this::buildLocation);
     }
 
+    @Override
+    public List<CwmsIdLocationKind> getLocationKinds(String nameRegex, String unitSystem, String datum, String officeId) {
```

Comment:
How about paging? Paging is not absolutely required but adding it later seems like it would require re-coding the Regi-side